examples/models/efficient_sam/efficient_sam_core/efficient_sam.py

```diff
@@ -0,0 +1,316 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Could you cite the source of this file, as well as the encoder/decoder?

Reply:
Could you also highlight if there are any modifications to make it exportable to ExecuTorch?

Reply:
Yes, of course